INFO     1.txt:my_log.py:27 *********登录接口开始测试*******
INFO     1.txt:my_log.py:27 {'name': '不输入用户名', 'requests': {'method': 'get', 'headers': {'Authorization': 'Basic aW9tczppb21z'}, 'data': {'username': None, 'password': '7G8EqAsTPjxcn4Mmn7LSpQ==', 'grant_type': 'password'}}, 'except': {'username': None, 'msg': 'Bad credentials'}}
INFO     1.txt:my_log.py:27 返回数据"Bad credentials"与数据断言"Bad credentials"一致，断言通过
INFO     1.txt:my_log.py:27 ******登录接口测试结束*******